### PR TITLE
[IMP] {planning}: add demo data

### PR DESCRIPTION
**Pull request by cedricpri**
Some demo data have been added on the planning module.

task-2677949
See odoo/enteprise#22536

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/data/hr_holidays_demo.xml [hunk 1]
```diff
@@ -438,5 +438,14 @@
...
     </record>
 
+    <!-- Public time off -->
+    <record id="resource_public_time_off_1" model="resource.calendar.leaves">
+        <field name="name">Public Time Off</field>
+        <field name="company_id" ref="base.main_company"/>
+        <field name="calendar_id" ref="resource.resource_calendar_std"/>
+        <field name="date_from" eval="(datetime.today() + relativedelta(months=+1)).strftime('%Y-%m-%d 07:00:00')"></field>
+        <field name="date_to" eval="(datetime.today() + relativedelta(months=+1)).strftime('%Y-%m-%d 16:00:00')"></field>
+    </record>
```
Comment (xavierbol) [inline addons/hr_holidays/data/hr_holidays_demo.xml:448]: If it is just for planning app, then we could add this demo data in planning_holidays module for instance. What do you think? 🙂

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I have made a comment. 🙂

**Review by IT-Ideas — APPROVED** [review]
Thanks for your work 🙂

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**